lib/active_model/serializer/adapter.rb

```diff
@@ -69,7 +72,8 @@ def lookup(adapter)
         # @api private
         def find_by_name(adapter_name)
           adapter_name = adapter_name.to_s.classify.tr('API', 'Api')
-          ActiveModel::Serializer::Adapter.const_get(adapter_name.to_sym) or # rubocop:disable Style/AndOr
+          ActiveModel::Serializer::Adapter.const_get(adapter_name.to_sym) ||
+            ActiveModel::Serializer::Adapter.const_get(adapter_name.pluralize.to_sym) or # rubocop:disable Style/AndOr
```

Comment:
the const_get breaks with the pluralize.. so either back to `safe_constantize` or singularize all adapter names when registering and looking up.. sigh